examples/echo_server/include/ethernet_config/ethernet_config.h

```diff
@@ -102,55 +102,49 @@ static inline void net_virt_mac_addrs(char *pd_name, uint64_t macs[NUM_NETWORK_C
+        *tx_queue_capacity = NET_TX_QUEUE_CAPACITY_CLI1;
     }
 }
 
-static inline void net_copy_queue_size(char *pd_name, size_t *cli_queue_size, size_t *virt_queue_size)
+static inline void net_copy_queue_capacity(char *pd_name, size_t *cli_queue_capacity, size_t *virt_queue_capacity)
 {
     if (!sddf_strcmp(pd_name, NET_COPY0_NAME)) {
-        *cli_queue_size = NET_RX_QUEUE_SIZE_CLI0;
-        *virt_queue_size = NET_RX_QUEUE_SIZE_COPY0;
+        *cli_queue_capacity = NET_RX_QUEUE_CAPACITY_CLI0;
+        *virt_queue_capacity = NET_RX_QUEUE_CAPACITY_COPY0;
     } else if (!sddf_strcmp(pd_name, NET_COPY1_NAME)) {
-        *cli_queue_size = NET_RX_QUEUE_SIZE_CLI1;
-        *virt_queue_size = NET_RX_QUEUE_SIZE_COPY1;
+        *cli_queue_capacity = NET_RX_QUEUE_CAPACITY_CLI1;
+        *virt_queue_capacity = NET_RX_QUEUE_CAPACITY_COPY1;
     }
 }
 
 typedef struct net_queue_info {
     net_queue_t *free;
     net_queue_t *active;
-    size_t size;
+    size_t capacity;
 } net_queue_info_t;
 
 static inline void net_virt_queue_info(char *pd_name, net_queue_t *cli0_free, net_queue_t *cli0_active,
                                        net_queue_info_t ret[NUM_NETWORK_CLIENTS])
 {
     if (!sddf_strcmp(pd_name, NET_VIRT_RX_NAME)) {
-        ret[0] = (net_queue_info_t) {
-            .free = cli0_free, .active = cli0_active, .size = NET_RX_QUEUE_SIZE_COPY0
-        };
-        ret[1] = (net_queue_info_t) {
-            .free = (net_queue_t *)((uintptr_t)cli0_free + 2 * NET_DATA_REGION_SIZE),
-            .active = (net_queue_t *)((uintptr_t)cli0_active + 2 * NET_DATA_REGION_SIZE),
-            .size = NET_RX_QUEUE_SIZE_COPY1
-        };
+        ret[0] =
+            (net_queue_info_t) { .free = cli0_free, .active = cli0_active, .capacity = NET_RX_QUEUE_CAPACITY_COPY0 };
+        ret[1] = (net_queue_info_t) { .free = (net_queue_t *)((uintptr_t)cli0_free + 2 * NET_DATA_REGION_SIZE),
+                                      .active = (net_queue_t *)((uintptr_t)cli0_active + 2 * NET_DATA_REGION_SIZE),
+                                      .capacity = NET_RX_QUEUE_CAPACITY_COPY1 };
     } else if (!sddf_strcmp(pd_name, NET_VIRT_TX_NAME)) {
-        ret[0] = (net_queue_info_t) {
-            .free = cli0_free, .active = cli0_active, .size = NET_TX_QUEUE_SIZE_CLI0
-        };
-        ret[1] = (net_queue_info_t) {
-            .free = (net_queue_t *)((uintptr_t)cli0_free + 2 * NET_DATA_REGION_SIZE),
-            .active = (net_queue_t *)((uintptr_t)cli0_active + 2 * NET_DATA_REGION_SIZE),
-            .size = NET_TX_QUEUE_SIZE_CLI1
-        };
+        ret[0] =
+            (net_queue_info_t) { .free = cli0_free, .active = cli0_active, .capacity = NET_TX_QUEUE_CAPACITY_CLI0 };
+        ret[1] = (net_queue_info_t) { .free = (net_queue_t *)((uintptr_t)cli0_free + 2 * NET_DATA_REGION_SIZE),
+                                      .active = (net_queue_t *)((uintptr_t)cli0_active + 2 * NET_DATA_REGION_SIZE),
```

Comment:
Should these ones be called size or capacity? In the driver I noticed the data regions are referred to with capacity.

Reply:
No, this is not the size of a queue, this is the size of the data region which is used for free and active queues (it is actually much bigger than the size of the queue). I mentioned in the description, I did not change the term size if it was used to describe the size of a memory region. There queue sits in that memory region, and actually only takes up a fraction of that size.

Reply:
Yes sorry, got confused with the serial drivers. Thought there was a change with the network drivers but there is not.